### PR TITLE
chore(web): change license wording and other things

### PR DESCRIPTION
**Pull request by alextran1502**
This PR changes 

- Changed the license wording. 
- Added the supporter badge.
- Added options to hide the buy button and the supporter badge.
- Buy button is only shown after 14 days of account age.

### Hide buy button options
image

#### After activation
image

#### User settings 
image

### CODE REVIEW

File: web/src/lib/components/shared-components/side-bar/purchase-info.svelte [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        <br />
+        <p>
+          {$t('purchase_panel_info_2')}
+        </p>
+      </div>
+      <div class="mt-3 flex flex-col gap-1">
+        <Button fullwidth on:click={openPurchaseModal}>{$t('purchase_button_buy_immich')}</Button>
+        <hr class="my-2" />
+        <Button size="sm" fullwidth on:click={() => hideButton(true)}>{$t('purchase_button_never_show_again')}</Button>
+        <Button size="sm" fullwidth on:click={() => hideButton(false)}>{$t('purchase_button_reminder')}</Button>
```
Comment (michelheusschen) [inline web/src/lib/components/shared-components/side-bar/purchase-info.svelte:171]: What do you think of this button design? 

image

Comment (alextran1502) [reply web/src/lib/components/shared-components/side-bar/purchase-info.svelte:171]: This design is also fine with me

Comment (michelheusschen) [reply web/src/lib/components/shared-components/side-bar/purchase-info.svelte:171]: Alright, I'll push those changes

File: web/src/routes/admin/system-settings/+page.svelte [hunk 2]
```diff
@@ -92,6 +92,7 @@
       subtitle: $t('admin.library_settings_description'),
       key: 'external-library',
     },
+
```
Comment (zackpollard) [inline web/src/routes/admin/system-settings/+page.svelte:95]: Random newline? :stuck_out_tongue:

**Review by zackpollard — APPROVED** [review]

**Pull request merged**